var/spack/repos/builtin/packages/gtkplus/package.py

```diff
@@ -29,10 +29,13 @@ class Gtkplus(Package):
     """The GTK+ 2 package contains libraries used for creating graphical user
        interfaces for applications."""
     homepage = "http://www.gtk.org"
+    url = "http://ftp.gnome.org/pub/gnome/sources/gtk+/2.24/gtk+-2.24.31.tar.xz"
 
     version(
-        '2.24.25', '612350704dd3aacb95355a4981930c6f',
-        url="http://ftp.gnome.org/pub/gnome/sources/gtk+/2.24/gtk+-2.24.25.tar.xz")
+        '2.24.31',
+        '68c1922732c7efc08df4656a5366dcc3afdc8791513400dac276009b40954658')
+    version(
+        '2.24.25', '612350704dd3aacb95355a4981930c6f')
```

Comment:
You can probably just put these on a single line. We exclude them from the line length checks anyway. Also, the longer hash is killing my OCD, but we do support it 😛